examples/step-67/step-67.cc

```diff
@@ -1717,8 +1717,7 @@ namespace Euler_DG
         // speed only on the valid cells of a cell batch.
         for (unsigned int v = 0; v < data.n_active_entries_per_cell_batch(cell);
              ++v)
-          for (unsigned int d = 0; d < 3; ++d)
-            max_transport = std::max(max_transport, local_max[v]);
+          max_transport = std::max(max_transport, local_max[v]);
```

Comment:
You can't ever be sure whether these modern processors do computations right. Better be safe and run things three times!